src/components/molecules/ZopaFooter/styles/Heading.tsx

```diff
@@ -1,6 +1,6 @@
 import styled from 'styled-components';
 import ZopaHeading from '../../../atoms/Heading/Heading';
 
-export const Heading = styled(ZopaHeading).attrs({ as: 'h3', size: 'h6' })`
+export const Heading = styled(ZopaHeading).attrs({ as: 'p', size: 'h6' })`
```

Comment:
I think this styled component should wrap the `Text` component as the `Heading` should not allow the attribute `as` as `p`